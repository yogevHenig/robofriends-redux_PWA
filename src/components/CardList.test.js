import React from 'react';
import CardList from './CardList';
import renderer from 'react-test-renderer';


it('expect to render card component', () => {

	const mockRobots = [
		{
			id:1,
			name: 'john snow',
			username:'johnjohn',
			email: 'john.gmail.com',
		}
	]

	const wrapper = renderer.create(<CardList robots={mockRobots} />).toJSON();
	expect(wrapper).toMatchSnapshot();
})