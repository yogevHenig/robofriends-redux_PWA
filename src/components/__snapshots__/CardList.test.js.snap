// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
<div>
  <div
    className="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/1?size=200x200"
    />
    <div>
      <h2>
        john snow
      </h2>
      <p>
        john.gmail.com
      </p>
    </div>
  </div>
</div>
`;
